pkg/ovsdb/transact.go

```diff
@@ -1318,20 +1322,78 @@ func etcdGetByWhere(txn *Transaction, ovsOp *libovsdb.Operation, ovsResult *libo
 	return nil
 }
 
-func etcdPutRow(txn *Transaction, key *common.Key, row *map[string]interface{}) error {
+func etcdCreateRow(txn *Transaction, k *common.Key, row *map[string]interface{}) error {
+	key := k.String()
 	val, err := makeValue(row)
 	if err != nil {
 		return err
 	}
 
-	etcdOp := clientv3.OpPut(key.String(), val)
+	etcdOp := clientv3.OpPut(key, val)
 	txn.etcd.Then = append(txn.etcd.Then, etcdOp)
+
+	etcdEvent := clientv3.Event{
+		Type: mvccpb.PUT,
+		Kv: &mvccpb.KeyValue{
+			Key:     []byte(key),
+			Value:   []byte(val),
+			Version: 1,
```

Comment:
you need to add  CreateRevision: and  ModRevision: (maybe copy from the eetcd response)
(they should be the same)

Reply:
I am just using values of 1 and 2

Reply:
it's OK